src/components/calculator.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import calculate from '../logic/calculate';
+
+function Calculator() {
+  const [state, setState] = useState({
+    total: '0',
+    next: null,
+    operation: null,
+  });
```

Comment:
- Please you are using ES6 Javascript, Kindly use ES class just like what you did below (in line 11).
- Please, After implementing the change make sure that your App keeps working as I checked